public/stylesheets/agent-home-style.css

```
@@ -7,4 +7,23 @@ button {
     border: none;
     border-radius: 5px;
     background-color: #3296ff;
-}
\ No newline at end of file
+}
+ 
+button:hover{
+    background-color: #267cd8;
+}
+ 
+h1{
+    font-size: 75px;
+}
+ 
+.project-picture{
+    display: flex;
+    justify-content: space-around;
+    flex-direction: column;
+ 
+}
+ 
+input:hover{
+    outline: 2px solid grey
```

Yeah yeah im being nitpicky wit da formatting.
jack try not to be an asshole challenge (IMPOSSIBLE)
```suggestion
button:hover {
    background-color: #267cd8;
}
 
h1 {
    font-size: 75px;
}
 
.project-picture {
    display: flex;
    justify-content: space-around;
    flex-direction: column;
}
 
input:hover {
    outline: 2px solid grey;
```